### PR TITLE
Refactor backend eslint rules

### PR DESCRIPTION
**Pull request by vgoh1**
# Description 
Refactoring these rules in the backend:
```
"camelcase": "off",
"consistent-return": "off",
"no-shadow": "off",
```

# Type of change 
- [ ] Bug fix
- [ ] New feature
- [x] Refactoring
- [ ] Documentation 

# Problem
Rules were suppressed with the addition of airbnb style guide. This is to refactor them so the rules don't have to be suppressed.

# Solution description
Self-explanatory.
I didn't touch `"prefer-destructuring": "off",` because I couldn't figure out how to destructure it.
`"no-plusplus": "off",` was decided to stay in a previous PR#208

# Tests 
I clicked around on everything and it looked like everything still worked. Feel free to click through as well to double check.

# Relevant document/ link (if any) 
Google relevant eslint rules if interested.

# Risk
Identity the risk level (low, medium, high) and indicate which component(s) will be affected if bugs exist.
High, all of backend

### CODE REVIEW

**Review by jfan0010 — APPROVED** [review]
LGTM

**Review by ssha0054 — APPROVED** [review]
Looks good!

File: api/src/middleware/auth.js [hunk 1]
```diff
@@ -37,16 +37,16 @@ const cookieJwtAuth = (req, res, next) => {
       const { team } = jwt.verify(refreshToken, process.env.JWT_SECRET_REFRESH_TOKEN || 'JWT_SECRET_REFRESH_TOKEN');
 
       // re-gernerate the accessToken
-      const accessToken = jwt.sign({ team }, process.env.JWT_SECRET_ACCESS_TOKEN || 'JWT_SECRET_ACCESS_TOKEN', {
+      const newAccessToken = jwt.sign({ team }, process.env.JWT_SECRET_ACCESS_TOKEN || 'JWT_SECRET_ACCESS_TOKEN', {
```
Comment (aksh0001) [inline api/src/middleware/auth.js:40]: could you fix the L39 typo too?

**Review by aksh0001 — APPROVED** [review]
LGTM, approval.

File: api/src/middleware/auth.js (hunk 1 above)
Comment (vgoh1) [reply api/src/middleware/auth.js:40]: Done

**Pull request merged**